docs/installation-deb.md

```diff
@@ -4,11 +4,9 @@
 
 We'll go over these in more detail below.
 
-* Cassandra. We run and recommend 3.0.8 .
+* Cassandra. We run and recommend 3.X .
```

Comment:
can't we be more specific? 3.x includes old versions that don't have some things we need (like TWCS)